js/app.js

```diff
@@ -33,6 +33,11 @@
         reloadPageLayout();
     }
 
+    var style = document.createElement('style');
+    style.type = 'text/css';
+    style.innerText = `.client_channels_list_container { display: auto; transition: margin 250ms cubic-bezier(.2,.8,.5,1),top 0s linear 0s,left 250ms ease-in-out; }`;
```

Comment:
@jrschumacher Thanks for the PR. Sorry for late reply, i've been busy at work.
Maybe we can just use single quote instead of backtick (just to make the code uniform) ?